app/customer.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from app.car import Car
+
+
+class Customer:
+    def __init__(
+        self, name: str, products: dict, location: list, money: int, car: Car
+    ) -> None:
+        self._name = name
+        self._products = products
+        self._location = location
+        self._money = money
+        self._car = car
+
+    @classmethod
+    def create_customer(cls, customer_info: dict) -> Customer:
+        return cls(
+            customer_info["name"],
+            customer_info["product_cart"],
+            customer_info["location"],
+            customer_info["money"],
+            Car.create_car(customer_info["car"]),
+        )

```

Comment:
The `create_customer` method assumes that the dictionary `customer_info` will always contain the keys 'name', 'product_cart', 'location', 'money', and 'car'. It would be safer to add error handling to manage cases where these keys might be missing, such as using `customer_info.get('name')` with a default value or raising an informative exception.